guides/rules/no-broken-super-chain.md

```diff
@@ -0,0 +1,25 @@
+# Don't break the super chain
+
+## Rule name: `no-broken-super-chain`
+
+If you are overriding the `init` lifecycle hook in Ember modules like Component, Mixin, etc. it is necessary that you include a call to `_super`
```

Comment:
Nitpick: Should likely say "Ember classes" instead of "Ember modules". Also, needs a period at the end of the sentence.